generators/server/templates/pom.xml.ejs

```diff
@@ -363,6 +363,18 @@
             <artifactId>liquibase-mssql</artifactId>
         </dependency>
         <%_ } _%>
+        <dependency>
```

Comment:
@DanielFran should I move the JUnit 5 dependency to https://github.com/jhipster/jhipster?